FLEDGE_extended_PA_reporting.md

```diff
@@ -204,6 +204,8 @@ Where `signalBucket` and `signalValue` is a dictionary which consists of:
     * 6: indicates seller rejected bid because “Creative Filtered - Language Exclusions”
     * 7: indicates seller rejected bid because “Creative Filtered - Category Exclusions”
     * 8: indicates seller rejected bid because "Creative Filtered - Did Not Meet The K-anonymity Threshold"
+    * 9: indicates bid produced by generateBid() was rejected because it failed a currency check
+    * 10: indicates bid passed through or altered by scoreAd() was rejected because it failed a currency check
```

Comment:
```suggestion
    * 9: indicates bid produced by `generateBid()` was rejected because it failed a currency check
    * 10: indicates bid passed through or altered by `scoreAd()` was rejected because it failed a currency check
```

Reply:
Done.